PROShine/Views/MapView.xaml.cs

```diff
@@ -138,9 +138,16 @@ private void MapCanvas_MouseRightButtonUp(object sender, MouseButtonEventArgs e)
             }
         }
 
+        private void MapCanvas_MouseEnter(object sender, MouseEventArgs e)
+        {
+            FloatingTip.IsOpen = true;
+            _selectedRectangle.Visibility = Visibility.Visible;
+        }

```

Comment:
I'm pretty sure this was already merged in a previous pull request.
Make sure you merge or rebase my changes before opening a pull request to prevent duplicates like this.

Reply:
Yeah, sorry,